src/pages/ListScholarships.jsx

```diff
@@ -22,6 +22,7 @@ const drawerWidth = 360;
 function ListScholarships() {
   const { t } = useTranslation();
   const [drawerOpen, setDrawerOpen] = useState(false);
+  const [filterCount, setFilterCount] = useState(0);
```

Comment:
This doesn't pick up filters already specified in the URL. E.g. no number appears if you go to https://s4us-pr-914.onrender.com/scholarships?maxAmount=300.

Just use `useQueryParams` in `FilterBar` and calculate the number of filters applied from there.